### ISSUE TITLE
Impact of Disabling Sweep Predictions

### ISSUE THREAD

**Issue by dzeego** [MIC-DKFZ/nnDetection#295]
Hello @mibaumgartner,

I've got a quick question please, what is the impact of setting `run_prediction=False` in the function _sweep() on the generation of the inference plan?
Thank you in advance!

**Comment by partha-ghosh** [comment]
Hi @dzeego,
The `–-sweep` option tells nnDetection to look for the best hyparameters for inference by empirically evaluating them on the validation set.  If you set `run_prediction=False`, predictions on the validation set are disabled, meaning hyperparameter optimization will not occur.

Best,
Partha
